carp.client.core/src/commonMain/kotlin/dk/cachet/carp/client/domain/data/DataListener.kt

```diff
@@ -2,28 +2,45 @@ package dk.cachet.carp.client.domain.data
 
 import dk.cachet.carp.common.data.Data
 import dk.cachet.carp.common.data.DataType
-import dk.cachet.carp.protocols.domain.devices.AnyDeviceDescriptor
 import dk.cachet.carp.protocols.domain.devices.DeviceRegistration
-import kotlin.reflect.KClass
+import dk.cachet.carp.protocols.domain.devices.DeviceType
 
 
 /**
- * Allows subscribing to [Data] of requested [DataType]s on a master device and connected devices.
+ * Allows subscribing to [Data] of requested [DataType]s on a master device and connected devices
```

Comment:
I know this PRs focus is on internal/external sensors. But will the `DataListener` also be able to subscribe to data coming from survey answers? Either we should start adressing the future need for collecting survey data or create an issue.

Reply:
All data is treated equally. Will update documentation to state _"Allows subscribing to [Data] (e.g., sensors, surveys)"_ to clarify.